src/nocks/github.test.ts

```diff
@@ -0,0 +1,44 @@
+// nocks/github.test.ts
+import nock from 'nock';
+// import { MockAgent, setGlobalDispatcher } from 'undici';
```

Comment:
remove commented-out code?

Reply:
urgh! I really thought I'd removed that..